WTWSMS/history/titles/d_cilicia.txt

```diff
@@ -13,6 +13,14 @@
 500.1.1={
 	holder=6860
 }
+524.1.1={
+    holder=1310010
+    liege="k_cilicia"
+}
+527.1.1={
+    holder=0
+	liege="k_cilicia"
+}
```

Comment:
But here you are dispossessing the character that holds the kingdom of Cilicia.

Reply:
I'll give it to the king of Cilicia then

Reply:
Solved